lib/mssql.js

```diff
@@ -423,6 +423,50 @@ function dateToMsSql(val) {
   }
 }
 
+function escape(val) {
+  if (val === undefined || val === null) {
+    return 'NULL';
+  }
+
+  switch (typeof val) {
+    case 'boolean':
+      return (val) ? 1 : 0;
+    case 'number':
+      return val + '';
+  }
+
+  if (typeof val === 'object') {
+    val = (typeof val.toISOString === 'function')
+      ? val.toISOString()
+      : val.toString();
+  }
+
+  val = val.replace(/[\0\n\r\b\t\\\'\"\x1a]/g, function (s) {
+    switch (s) {
+      case "\0":
+        return "\\0";
+      case "\n":
+        return "\\n";
+      case "\r":
+        return "\\r";
+      case "\b":
+        return "\\b";
+      case "\t":
+        return "\\t";
+      case "\x1a":
+        return "\\Z";
+      case "\'":
+        return "''"; // For sql server, double quote
+      case "\"":
+        return s; // For oracle
+      default:
+        return "\\" + s;
+    }
+  });
+  // return "q'#"+val+"#'";
+  return "'" + val + "'";
+}
```

Comment:
Until we ditch out string concatenation in favour of parameterized commands/queries: it would be nice to move this escape function to loopback-connector-sql so that there is only one instance of this code shared by all SQL connectors.